lit/docs/guides/git.lit

```diff
@@ -139,3 +139,230 @@
+
+    In addition to the branch pipeline template, we'll also need a pipeline to
+    track the list of branches and set a pipeline for each one.
+
+    To track the list of branches in a repository, we can use
+    \link{aoldershaw/git-branches-resource}{https://github.com/aoldershaw/git-branches-resource}.
+    This \reference{resource-types}{\code{resource_type}} emits a new
+    \reference{resource-versions}{resource version} whenever a branch is created or
+    deleted. It also lets us filter the list of branches by a regular expression.
+    In this case, let's assume our feature branches match the regular expression
+    \code{feature/.*}.
+
+    Below is the full pipeline config for this tracker pipeline:
+
+    \link{\code{examples/pipelines/multi-branch/tracker.yml}}{https://github.com/aoldershaw/examples/blob/master/pipelines/multi-branch/tracker.yml}
+    \codeblock{yaml}{{{
+      resource_types:
+      - name: git-branches
+        type: registry-image
+        source:
+          repository: aoldershaw/git-branches-resource
+
+      resources:
+      - name: feature-branches
+        type: git-branches
+        source:
+          uri: https://github.com/concourse/examples
+          branch_regex: 'feature/(?P<feature>.*)'
+
+      - name: examples
+        type: git
+        source:
+          uri: https://github.com/concourse/examples
+
+      jobs:
+      - name: set-feature-pipelines
+        plan:
+        - in_parallel:
+          - get: feature-branches
+            trigger: true
+          - get: examples
+        - load_var: branches
+          file: feature-branches/branches.json
+        - across:
+          - var: branch
+            values: ((.:branches))
+          set_pipeline: dev
+          file: examples/pipelines/multi-branch/template.yml
+          instance_vars: {feature: ((.:branch.groups.feature))}
+          vars: {branch: ((.:branch.name))}
+    }}}
+
+    We set each pipeline as an \reference{instanced-pipelines}{instanced
+    pipeline} - this will result in Concourse grouping all of the related
+    \code{dev} pipelines in the UI.
+  }
+
+  \section{
+    \title{Cleaning Up Old Workspaces}{multi-branch-cleaning-up}
+
+    With the setup described in \reference{multi-branch-tracking-branches},
+    Concourse will automatically archive any pipelines for branches that get
+    removed. However, Concourse doesn't know that it should destroy Terraform
+    workspaces when a branch is removed. To accomplish this, we can yet again
+    make use of the \link{Terraform resource}{https://github.com/ljfranklin/terraform-resource}
+    to destroy these workspaces. We'll add another job to the tracker pipeline
+    that figures out which workspaces don't belong to an active branch and
+    destroy them.
+
+    \link{\code{examples/pipelines/multi-branch/tracker.yml}}{https://github.com/aoldershaw/examples/blob/master/pipelines/multi-branch/tracker.yml}
+    \codeblock{yaml}{{{
```

Comment:
I can see you wanted to omit some stuff here, explains why you didn't use `\remote-codeblock{}` 👍